src/WebJobs.Extensions.EasyTables/Bindings/CompositeBindingProvider.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Host.Bindings
```

Comment:
Put a TODO in the code - we'll need to refactor the binding to remove the need for a copy of this interface. The interface wasn't designed for this - we'll want to put the right pattern in place for people to follow.

Can you log a bug in this repo to do this? BTW - I realize you did what made sense in the absence of any established guidance :)